### DISCUSSION TITLE
Styled Datepicker

### DISCUSSION THREAD

**Discussion by dlongodev** [skeletonlabs/skeleton#1474] [Ideas]
Any plans for styled Datepicker component or maybe any Select Dropdowns that are styled other than native dropdown (sort of like Material Design approach for selects)?

**Comment by endigo9740** [comment]
Native or third party is our recommendation for now. However we did have a feature request for this recently:
- https://github.com/skeletonlabs/skeleton/issues/1452

Long term we would love to have this, but it may be a while before we can implement due to the complexity and other priorities on the table right now.
